src/desktop/apps/artwork/lib/inquire.coffee

```diff
@@ -4,7 +4,7 @@ Artwork = require '../../../models/artwork.coffee'
 ArtworkInquiry = require '../../../models/artwork_inquiry.coffee'
 openInquiryQuestionnaireFor = require '../../../components/inquiry_questionnaire/index.coffee'
 
-module.exports = (id) ->
+module.exports = (id, ask_specialist = false) ->
```

Comment:
@sweir27 - passing this `ask_specialist` as on optional variable here to pass through to the `openInquiryQuestionnaireFor`. Does look cleaner I think. Thanks for pointing me to that option. 

How does that look to you?